client/src/components/atoms/ImagePicker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from "react";
+import ImageList from "./ImageList";
+import Row from "react-bootstrap/Row";
+
+interface Props {
+  onImageChange(url: string): void;
+  images: string[];
+  selected: string;
+  isErroneous: boolean;
+}
+
+const ImagePicker: FunctionComponent<Props> = (props: Props) => {
+  const { selected, images, onImageChange, isErroneous } = props;
+  return (
+    <div className="imagepicker">
+      <Row className="toprow">
+        <div className="imagepicker-title">Image</div>
```

Comment:
Is it possible to remove the title "Image" and the error text message "Please select an image"?
We're planning to wrap all input components (ImageList, TextField, etc.) in [FormItem](https://www.notion.so/uwblueprintexecs/34a44be943164278bfbf43de35e4ef7b?v=448f1b24885146a7b264dbf09eb60be1) which will add the header and error message.

Sorry, I should've been more clear in ur ticket description!